### PR TITLE
test(cadl): initial sdk for adp

### PR DESCRIPTION
**Pull request by archerzz**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by chunyu3 — COMMENTED** [review]
cadl files should not be commit to sdk repo. So is it a temporary solution, it will be removed in future?

**Review by lirenhe — COMMENTED** [review]
Could you help to confirm this folder structure?  sdk/adp/Azure.AutonomousDevelopmentPlatform, shall we call it 'Azure.Developer.AutonomousDevelopmentPlatform,'

File: sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/cadl.json [hunk 1]
```diff
@@ -0,0 +1,3539 @@
+{
```
Comment (lirenhe) [inline sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/cadl.json:1]: shall we include this file?

File: sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/Configuration.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (lirenhe) [inline sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/Configuration.json:1]: Same for this one

File: sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/cadl.json (hunk 1 above)
Comment (archerzz) [reply sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/cadl.json]: deleted

File: sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/Configuration.json (hunk 2 above)
Comment (archerzz) [reply sdk/adp/Azure.AutonomousDevelopmentPlatform/Generated/Configuration.json]: deleted

**Comment by archerzz** [comment]
close, better to create a new PR.

**Pull request closed without merging**